pkg/apps/application-list.jsx

```diff
@@ -69,7 +69,7 @@ class ApplicationRow extends React.Component {
                         {comp.summary}
                         <div className="alert alert-danger alert-dismissable">
                             <button className="close"
-                                    onClick={left_click(() => { this.setState({ error: null }) })}>
+                                    onClick={left_click(() => { this.setState({ error: null }); })}>
```

Comment:
IMHO this is a bit over the top. I think this should rather drop the unnecessary { } *and* the semicolon for a single-line function call. I. e. `() => this.setState(...)`.